main.nf

```diff
@@ -162,20 +163,23 @@ workflow {
   ch_is_staging = Channel.value(is_staging)
 
   // if (params.force) {
-  //   reset_processing(center_map_synid)
+  //   reset_processing(center_map_synid)]
   //   reset_processing.out.view()
   // }
+
   if (params.process_type == "only_validate") {
     validate_data(ch_project_id, ch_center)
     // validate_data.out.view()
   } else if (params.process_type == "maf_process") {
     // Call the function
-    process_maf_helper(params.maf_centers, ch_project_id, maf_center_list, params.create_new_maf_db)
+    sync_done = sync_staging_table_with_production(ch_is_staging, params.sync_staging_table_with_production)
```

Comment:
I think for this method you just have to remove `sync_done` and then change `sync_done.out` to be `sync_staging_table_with_production.out` in the subsequent processes